src/index.html

```diff
@@ -6,9 +6,33 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#home"><img src="logo.343.png" alt="Logo"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a class="nav__link is-active" href="#apple">Apple</a></li>
+          <li class="nav__item"><a class="nav__link" href="#samsung">Samsung</a></li>
+          <li class="nav__item"><a class="nav__link" href="#smartphones">Smartphones</a></li>
+          <li class="nav__item">
+            <a class="nav__link"
+            href="#laptops&computers"
```

Comment:
It's recommended to avoid using special characters like '&' in href attribute. Replace '&' with 'and' or use a more descriptive value.

Reply:
done